internal/management/controller/instance_controller.go

```diff
@@ -741,51 +743,54 @@ func (r *InstanceReconciler) reconcilePoolers(
 	}()
 
 	var existsRole bool
-	row := tx.QueryRow(fmt.Sprintf("SELECT COUNT(*) > 0 FROM pg_catalog.pg_roles WHERE rolname = '%s'",
-		apiv1.PGBouncerPoolerUserName))
+	row := tx.QueryRow("SELECT COUNT(*) > 0 FROM pg_catalog.pg_roles WHERE rolname = $1",
+		apiv1.PGBouncerPoolerUserName)
 	err = row.Scan(&existsRole)
 	if err != nil {
 		return err
 	}
 	if !existsRole {
-		_, err := tx.Exec(fmt.Sprintf("CREATE ROLE %s WITH LOGIN", apiv1.PGBouncerPoolerUserName))
+		_, err := tx.Exec(fmt.Sprintf("CREATE ROLE %s WITH LOGIN", pgx.Identifier{apiv1.PGBouncerPoolerUserName}.Sanitize()))
 		if err != nil {
 			return err
 		}
-		_, err = tx.Exec(fmt.Sprintf("GRANT CONNECT ON DATABASE %s TO %s", dbName, apiv1.PGBouncerPoolerUserName))
+		_, err = tx.Exec(fmt.Sprintf("GRANT CONNECT ON DATABASE %s TO %s",
+			pgx.Identifier{dbName}.Sanitize(),
+			pgx.Identifier{apiv1.PGBouncerPoolerUserName}.Sanitize()))
 		if err != nil {
 			return err
 		}
 	}
 
 	var existsFunction bool
-	row = tx.QueryRow(fmt.Sprintf("SELECT COUNT(*) > 0 FROM pg_catalog.pg_proc WHERE proname='%s' and prosrc='%s'",
+	row = tx.QueryRow("SELECT COUNT(*) > 0 FROM pg_catalog.pg_proc WHERE proname = $1 AND prosrc = $2",
 		userSearchFunctionName,
-		userSearchFunction))
+		userSearchFunction)
 	err = row.Scan(&existsFunction)
```

Comment:
We should explicitly sanitize these values using the previous fmt.sprintf, not using the values from the QueryRow() format, even if they supposed to sanitize , better to be explicit